Application Lifecycle Management/instances-api.md

```diff
@@ -1120,68 +1402,6 @@ In this sample request, the instance is tagged and scheduled to terminate at a g
-| box.updated | string | Date of the last update. |
-| box.description | string | Box description. |
-| box.service | string | Required. Can be one of these types: Linux Compute, Windows Compute and CloudFormation Service. |
-| box.tags | array | Box tags. |
-| box.variables | array | 	List of box variables, each variable object contains the parameters: type , name and value. |
-| box.created | string | Creation date. |
-| box.uri | string | Box uri. |
-| box.id | array | Box unique identifier. |
-| box.schema | string | Box schema uri. |
-| box.members | array | List of Box members. |
-| box.owner | string | Box owner. |
-| box.bindings | array | List of Box bindings. |
-| box.binding | object | Binding contained in the bindings list, each binding have a box and a name. |
-| box.icon | string | Box icon uri. |
-| box.events | array | 	List of Box events, there may be nine event lists: configure, dispose, install, post_configure, post_dispose, post_install, post_start, post_stop, start, and stop. |
-| box.event | object | Event contained in one of the event lists, each event object contains the parameters: url , upload_date , length and destination_path. |
-| box.name | string | Box name. |
-| policy_box | object | specific deployment policy for a provider |
-| policy_box.provider_id | string | provider id |
-| policy_box.automatic_updates | string | One of them: mayor, minor, patch, off |
-| policy_box.name | string | Policy box name |
-| policy_box.variables | array | List of deployment box policy variables |
-| policy_box.claims | array | List of deployment box policy claims |
-| policy_box.id | string | Deployment box policy id |
-| policy_box.schema | string | Deployment box policy schema |
-| policy_box.members | array | 	List of members sharing the deployment policy box |
-| policy_box.owner | string | Deployment box policy owner |
-| policy_box.readme | object | Readme file for the Deployment policy box |
-| uri | string | instance uri. |
-| state | string | Instance state, there are three possible states: processing , done and unavailable |
-| members | array | Instance members. |
-| owner | string | Instance owner. |
-| operation | string | Last operation, there are seven types of operations: deploy , shutdown , poweron , reinstall , reconfigure , terminate and terminate_service |
-| icon | string | Instance icon uri. |
-| id | array | Instance unique identifier. |
-| schema | string | Instance schema uri. |
-| lease | array | Schedules an instance with two parameters:<li>released. Is a true or false boolean value. False means that the operation scheduled on the instance is not executed yet.</li><li>expire. Specifies in UTC format YYYY-MM-DD HH:MM:SS.SSSSSS, the time and date for stopping an instance. It’s required only when an instance is set to terminate or shut down.</li><li>operation. Specifies an instance to stop with **shutdown** or **terminate**. When not scheduled, the instance is set to **alwayson**.</li> |
 
 ```
```

Comment:
Did you paste this block twice? (lines 1406-1535)
Remove it or comment it before, if it's a new example